pkgs/applications/audio/distrho/default.nix

```diff
@@ -1,16 +1,13 @@
 { stdenv, fetchgit, alsaLib, fftwSinglePrec, freetype, jack2
 , libxslt, lv2, pkgconfig, premake3, xlibs }:
 
-let
-  rev = "3bfddf7f";
-in
 stdenv.mkDerivation rec {
-  name = "distrho-${rev}";
+  name = "DISTRHO-Ports-git-2015-01-28";
```

Comment:
Thinking of it now, upper case should be avoided in names. http://nixos.org/nixpkgs/manual/#idm140737319157136